playbook.yml

```diff
@@ -0,0 +1,8 @@
+- hosts: all
+
+  tasks: 
+    - shell: echo hello world
+      sudo: yes
+    - shell: curl -IL tut.by
```

Comment:
Shell should only be used when piping, redirecting or chaining commands (and Ansible would be preferred for some of those!)